--- cli.js	2017-04-26 12:40:58 +0000
+++ cli.patched.js	2017-04-26 12:53:29 +0000
@@ -415,8 +415,10 @@
         // if we didn't find the template, keep the user-specified value so the error message is
         // useful
         return templatePath || env.opts.template;
     })();
+	
+	global.env = env;//FIX MOD russa: env seems to be local variable, but templates need access to global env!
 
     try {
         template = require(env.opts.template + '/publish');
     }
